cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -149,17 +147,22 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
       initQueries("CREATE USER jake SET PASSWORD 'abc123' CHANGE NOT REQUIRED",
         "CREATE USER user1 SET PASSWORD 'abc'", "CREATE USER user2 SET PASSWORD 'abc'", "CREATE USER user3 SET PASSWORD 'abc'")
       p("Roles can be created and managed using a set of Cypher administration commands executed against the `system` database.")
+      p("include::role-management-syntax.asciidoc[]")
       note {
-        p("""The role name `PUBLIC` is not permitted. This is a system-reserved role, and cannot be used.""")
+        p(
+          """There exists a special built-in role, `PUBLIC`, which is assigned to all users upon creation.
+            |This role cannot be dropped or revoked from any user, but its privileges may be modified.
+            |By default, it has the privilege to <<administration-security-administration-database-access, access>> the default database.
+            |""".stripMargin)
```

Comment:
I think this information is odd in an admonition. I am suggesting to move it into the section that describes built-in roles. See below.